src/test/regress/expected/bfv_partition_plans_optimizer.out

```diff
@@ -1,5 +1,43 @@
 create schema bfv_partition_plans;
 set search_path=bfv_partition_plans;
+SET client_min_messages = WARNING;
+create table t1 (a int, b int, c text, d int) with (appendoptimized=true, orientation=column) distributed randomly partition by range(a) ( start (1) end (100) every (1), default partition extra );
+create table t2 (a int, b int, c text, d int) with (appendoptimized=true, orientation=column) distributed randomly partition by range(a) ( start (1) end (100) every (1), default partition extra );
```

Comment:
Could you divide these line to add more readability?

Reply:
> Could you divide these line to add more readability?

done